components/ui-api-layer/docs/resource-quotas-status.md

```diff
@@ -0,0 +1,116 @@
+# ResourceQuotasStatus 
+
+[ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/) provides constraints that limit aggregate resource consumption per Namespace.
+
+The ResourceQuota values can be as follows:
+```bash
+Name:            kyma-default
+Resource         Used       Hard
+--------         ---        ---
+limits.memory    7348440Ki  10Gi
+requests.memory  3727576Ki  7Gi
+```
+The `used` values of the ResourceQuota can exceed the `hard` values and block any resource creation in the Environment.
+
+The ResourceQuotasStatus contains the flag and list of exceeded ResourceQuotas limits, together with a set of resources which exceed that limits. The value of the flag informs you if any ResourceQuota is exceeded in any possible way.
+
+ResourceQuotasStatus should detect whenever any ReplicaSet or StatefulSet in your Environment cannot progress because it's blocked by the ResourceQuota.
```

Comment:
`The ResourceQuotasStatus detects if any ReplicaSet or StatefulSet in your Environment is blocked by the ResourceQuota and cannot progress.`